### ISSUE TITLE
Implement views

### ISSUE THREAD

**Issue by anguslmm** [anguslmm/pullupsforpatriots#23]
Build views for each of the URLs we just implemented.

This means that you'll be changing `views.placeholder` to `views.nameoftheview` in both the `urls.py` files, which means you have to create a view for that in the `views.py` files, which means you'll have to create templates in the `templates` folders. I recommend you create simple templates, then create views, then change the url files.

For now, the templates can just be "This is the pagenamehere page." For the pages in the fundraiser app (i.e. dynamic pages), make sure the view gets the relevant data from the model, and passes it to the template, then try and do something simple. For example, maybe display the name of the Marine on the `marine/id` page, so we know the data is making it there.

Make sure you test your changes before submitting them. Run `python manage.py runserver $IP:$PORT`, then open the site and go to the page you just created a view for, and make sure it's displaying properly.

To avoid duplication of effort, when you start working on a view, please make a comment on this issue to let everyone know you've got that one. Do commits as you work, but don't submit a pull request until you've done all the views you're gonna do.

**Comment by caruiz1996** [comment]
working on homepage

**Comment by questornot** [comment]
quest- working on the ourcasue page

**Comment by questornot** [comment]
working on the OURCAUSE page

**Comment by anguslmm** [comment]
Hey guys, don't be afraid to push your branches as you work. That way your work is saved on this copy of the repo, and I can peak at stuff and answer questions from far away.

Just don't do a pull request until you're done.

**Comment by anguslmm** [comment]
I went ahead and implemented the marine/id page. You guys can feel free to look at what I did. Notice I used the generic views from the end of the Django tutorial instead of doing it the hard way like they have you do at the beginning.

Also, you're gonna wanna do a pull of my branch `marine-detail` and merge it into yours. That will bring the marine/id page into yours, and fix a couple things that would've given you problems. Also, I changed the Marines model to be named Marine.
